influx/databases.go

```diff
@@ -161,6 +151,11 @@ func (c *Client) DropRP(ctx context.Context, database string, rp string) error {
 	return nil
 }
 
+// AllMeasurements returns all measurements for a specific database
+func (c *Client) AllMeasurements(ctx context.Context, database string, limit, offset int) ([]chronograf.Measurement, error) {
```

Comment:
it seems like a function called `AllMeasurements` would effectively bypass limit and offset and thus drive `c.showMeasurements` to return all measurements. would this be more accurately called `Measurements` or `GetMeasurements`? it appears that that `AllDB` and `AllRP` return all of their respective data, whereas `AllMeasurements` takes an offset and limit and would only return _all_ if offset and limit were `0`, no?